### PR TITLE
Adds a Clown ERT

### PR DESCRIPTION
**Pull request by fluffe9911**
Honk

### CODE REVIEW

**Comment by swissloaf** [comment]
Good PR ! @wiki It has always been real !

**Comment by ynot01** [comment]
@fluffe9911 can you be more descriptive i.e. what does it do, what does it have, why is it called, what potential variants are there

File: code/modules/clothing/outfits/ert.dm [hunk 1]
```diff
@@ -304,3 +304,39 @@
...
+		/obj/item/reagent_containers/spray/waterflower = 1,
+		/obj/item/reagent_containers/food/snacks/grown/banana = 1,
+		/obj/item/instrument/bikehorn = 1,
+		)
+	implants = list(/obj/item/implant/sad_trombone)
+	back = /obj/item/storage/backpack/clown
+	box = /obj/item/storage/box/hug/survival
+	chameleon_extras = /obj/item/stamp/clown
+
+/datum/outfit/ert/clown/robust
```
Comment (alexkar598) [inline code/modules/clothing/outfits/ert.dm]: Call it /datum/outfit/ert/clown/leader

**Review by alexkar598 — DISMISSED** [review]

File: code/datums/ert.dm [hunk 2]
```diff
@@ -62,3 +62,12 @@
...
 	mission = "Clean up EVERYTHING."
 	polldesc = "a Nanotrasen Janitorial Response Team"
+
+/datum/ert/clown
+	roles = list(/datum/antagonist/ert/clown, /datum/antagonist/ert/clown/robust)
+	leader_role = /datum/antagonist/ert/clown/robust
+	teamsize = 7
+	opendoors = FALSE
+	rename_team = "HONK Squad"
+	mission = "Bring joy and happiness to all the crew the only way a clown can honk!"
```
Comment (Ktlwjec) [inline code/datums/ert.dm]: mission = "Bring joy and happiness to all the crew the only way a clown can. Honk!"

**Review by Ktlwjec — DISMISSED** [review]

File: code/datums/ert.dm (hunk 2 above)
Comment (alexkar598) [reply code/datums/ert.dm]: Canned clowns :thumbsup:

Comment (swissloaf) [reply code/datums/ert.dm]: still not Honk

**Comment by swissloaf** [comment]
> @fluffe9911 can you be more descriptive i.e. what does it do, what does it have, why is it called, what potential variants are there

job for wiki to create innit

**Comment by Amelia0010** [comment]
@swissloaf that's not how any of this works.

**Pull request merged**